tests/e2e/README.md

```diff
@@ -96,6 +96,20 @@ make push
 
 ```
 
+**Option 1.5:** Build your own images and use an in-cluster registry
+
+```
+# Set up env vars in .istiorc.mk
+cat .istiorc.mk
+LOCALREG=true
```

Comment:
Can use just do this automatically using make arguments?

something like "make run_registry_in_cluster=true test e2e_simple"?